File: M2/Ex1/Form1.vb
Public Class Form1
    Private Sub Label1_Click(sender As Object, e As EventArgs) Handles Label1.Click

    End Sub

    Private Sub Form1_Load(sender As Object, e As EventArgs) Handles MyBase.Load, MyBase.Click
        lblResposta.Text = "?"
    End Sub

    Private Sub BtnAdvinhe_Click(sender As Object, e As EventArgs) Handles btnAdvinhe.Click
        lblResposta.Text = "Clube da robótica"

    End Sub

    Private Sub BtnSair_Click(sender As Object, e As EventArgs) Handles btnSair.Click
        MessageBox.Show("Deseja Sair?", "Saída", MessageBoxButtons.YesNo, MessageBoxIcon.Warning)
    End Sub
End Class

File: M2/Ex2/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace Ex2
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void BtnOK_Click(object sender, EventArgs e)
        {
            MessageBox.Show("Olá " +txtInsere.Text+ " Bem vindo/a ao C#!");
        }

        private void BtnSair_Click(object sender, EventArgs e)
        {
            DialogResult left;
            left = MessageBox.Show("Deseja Sair?", "Saída", MessageBoxButtons.YesNo, MessageBoxIcon.Warning);
            if (left == DialogResult.Yes)
                Application.Exit();
        
        }

        private void BtnOK_KeyPress(object sender, KeyPressEventArgs e)
        {
          
        }
    }
}
 